test/client/data_test.rb

```diff
@@ -92,4 +92,11 @@ def test_data_update_custom_field
       assert_true result
     end
   end
+
+  def test_data_get_app_setting
```

Comment:
Yep.. I've encountered this problem as well. In fact, there are random times that the xmlrpc server will return anything in the content-type header. For CustomerHub I had to patch the xml-rpc client for Ruby to be able to handle issues like that. There was some discussion a long time ago on some forums about this, but the maintainer of the client refused to think it was a problem... which maybe it isn't :)